src/components/ChatApp/Settings/DevicesTab.react.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import Translate from '../../Translate/Translate.react';
+import TableComplex from '../../TableComplex/TableComplex.react';
+import SwipeableViews from 'react-swipeable-views';
+import MapContainer from '../../MapContainer/MapContainer.react';
+import PropTypes from 'prop-types';
+
+class DevicesTab extends Component {
```

Comment:
Use functional component here, as we do not need to declare it as a React component